test/datetime/invalid.test.js

```diff
@@ -39,7 +39,7 @@ test('Invalid tell you why', () => {
 });
 
 test('Invalid DateTimes return invalid Dates', () => {
-  expect(organic1.toJSDate().valueOf()).toBeFalsy();
+  expect(organic1.toJSDate().valueOf()).toBe(NaN);
```

Comment:
Surprised this works, actually.

Reply:
It didn't work initially, was fixed by https://github.com/facebook/jest/pull/4917, that's why I updated `jest`.